requirements.in

```diff
@@ -0,0 +1,14 @@
+Authlib
+cachetools
+elasticsearch
+elasticsearch-dsl
```

Comment:
think these never quite got used

Reply:
yeah... `grep.py` in master actually already imports from elasticsearch and elasticsearch_dsl so figured i'd add them here even if they're not used, if only to make the import work.